src/main/java/com/ericsson/ei/frontend/utils/WebControllerUtils.java

```diff
@@ -64,16 +64,16 @@
     public String getFrontEndServiceUrl() {
         String requestedUrl = null;
         String http = "http";
-        String path = "";
+        String contextPath = "";
         if (useSecureHttpFrontend) {
         	http = "https";
         }
 
         if (frontendContextPath != null && !frontendContextPath.isEmpty()) {
-        	path = ("/" + frontendContextPath).replace("//", "/");
+        	contextPath = ("/" + frontendContextPath).replace("//", "/");
         }
 
-        requestedUrl = String.format("%s://%s:%d%s", http, frontendServiceHost, frontendServicePort, path);
+        requestedUrl = String.format("%s://%s:%d%s", http, frontendServiceHost, frontendServicePort, contextPath);
```

Comment:
Maybe we should use a URL lib to join the url paths correctly? Then we don't need to manually add "/" and replace "//" etc. I think that is more safe.

Reply:
That is not my implementation and I think we keep it as it is for the moment, because the line before this it is doing something special handling of double "//" around contextPath variable,,, maybe a special Windows handling thing or something.

Reply:
Alright :)